package.json

```diff
@@ -72,12 +72,16 @@
     "precommit-test"
   ],
   "devDependencies": {
-    "appium-gulp-plugins": "^5.1.0",
-    "appium-test-support": "^1.1.0",
+    "@appium/gulp-plugins": "^6.0.0",
+    "@appium/eslint-config-appium": "^5.0.0",
+    "@appium/test-support": "^1.0.0",
     "@semantic-release/git": "^10.0.1",
-     "chai": "^4.1.2",
+    "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
-    "eslint-config-appium": "^4.0.1",
+    "eslint": "^7.32.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-mocha": "^9.0.0",
+    "eslint-plugin-promise": "^6.0.0",
     "gulp": "^4.0.0",
     "pre-commit": "^1.2.2",
     "semantic-release": "^19.0.2",
```

Comment:
wd should replaced with webdriverio as a part of this change and the corresponding e2e tests must be updated

Reply:
otheriwise we should disable failing e2e tests and anyway exclude wd from dependencies list